backend/src/rating/ranking.ts

```diff
@@ -0,0 +1,187 @@
+import type { youtube_v3 } from "googleapis/build/src/apis/youtube/v3";
```

Comment:
I would add a comment at the top of the file with a light weight explanation of how this ranking works and link to the doc